src/Essentials/src/MediaPicker/MediaPicker.android.cs

```diff
@@ -62,7 +62,7 @@ public async Task<FileResult> CaptureAsync(MediaPickerOptions options, bool phot
 				throw new FeatureNotSupportedException();
 
 			await Permissions.EnsureGrantedAsync<Permissions.Camera>();
-			await Permissions.EnsureGrantedAsync<Permissions.StorageWrite>();
```

Comment:
Was this sort of a mistake before having Write? or was Read introduced later?

Reply:
Do we not need write for _taking_ a picture/video now?

Reply:
Testing here now and seems not. We are using intents and file providers:
 - https://github.com/dotnet/maui/commit/add928f57b47c1b59beab2447a5cf224add75b50#diff-3bb30c6ffa6364915da9886b3a540d271b95d271bf9f9d71291ce77c67baa9d9
 - https://github.com/dotnet/maui/commit/add928f57b47c1b59beab2447a5cf224add75b50#diff-344a186cbba1897e7020b37df9065a3bb7f6dde1cbc88d70975620e6b6b80113R121

Reply:
According to the Android 11 docs, the Read and Write permissions do nothing: https://developer.android.com/about/versions/11/privacy/storage#permissions-target-11

> If your app targets Android 11, both the WRITE_EXTERNAL_STORAGE permission and the WRITE_MEDIA_STORAGE privileged permission no longer provide any additional access.

Also, having Write automatically adds Read, so maybe we always just needed that?

Reply:
Adding write back as I see other people do that... and also, rock the boat. Who knows, take a photo in Australia and the permissions are reversed.